.github/workflows/ci.yml

```
@@ -663,7 +663,7 @@ jobs:
         id: build-camunda-docker
         with:
           repository: camunda/camunda
-          version: SNAPSHOT
+          version: 8.7.0-SNAPSHOT
```

```suggestion
          version: 8.7-SNAPSHOT
```
wondering if we can omit the patch version here